css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
+    </header>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-2">
+        <img src="images/dress1.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/ysl-bag.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
```

Comment:
The different products should have some space between their images:

---

![image](https://user-images.githubusercontent.com/16143771/152214098-4d2e3dd8-9846-450b-bddc-f94d2fbfa78b.png)

---

![image](https://user-images.githubusercontent.com/16143771/152214138-bad6cb34-3c82-4251-a7b8-179a29cf9bc2.png)